### PR TITLE
[DetailsPage] Add context control

### PR DESCRIPTION
**Pull request by kyledurand**
### WHY are these changes introduced?

This adds a context control to our details page. Allowing quick comparison from the theme provider logo (present on the frame story page) and context control

## 

### How to 🎩

You just need to checkout the details page and make sure the context control looks good from within the nav on small screens, and in the top bar on larger screens

### 🎩 checklist

* [x] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
* [ ] Updated the component's `README.md` with documentation changes
* [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by tmlayton** [comment]
Styling is off on small screens, otherwise looks good!

Screen Shot 2020-01-16 at 9 11 23 PM

File: config/typescript/styles.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+declare module '*.scss' {
```
Comment (BPScott) [inline config/typescript/styles.d.ts]: This should already be covered by our ts config importing https://github.com/Shopify/sewing-kit/blob/master/packages/typescript-configs/definitions/styles.d.ts, do we need this too?

Comment (kyledurand) [reply config/typescript/styles.d.ts]: I was having weird typescript issues with this today. Seemed like I needed to add it but maybe there's just something wonky going on in this branch. The CI failure seems to confirm that

**Comment by danrosenthal** [comment]
Need anything else here to get this shipped?

Are you aware of the `Skip changelog` label? It allows that check to pass for changes we don't care about documenting.

**Comment by kyledurand** [comment]
> Need anything else here to get this shipped?

TS doesn't seem to like the addition of the scss file in this directory. Adding the module didn't seem to help either. @BPScott would you be down for pairing on a fix for that?

**Comment by BPScott** [comment]
Had a super speedy chat with Kyle - it was down to playground not being in the tsconfig.json and thus those files not picking up the project config.

As a bonus that file is now checked when you do `yarn run type-check`

**Pull request merged**